### DISCUSSION TITLE
Feedback on guide from WFP Digital Social protection

### DISCUSSION THREAD

**Discussion by sympaticus** [spdci/.github#10] [General]
Hi there, 
In terms of feedback, are you accepting comments/feedback until the end of August? And if so, in what modality? In-document / emails / what platform? 
Thank you 
Joe

**Comment by Apoorva265** [comment]
Hello Joe,

Thank you for your interest in providing feedback. Yes, we are indeed accepting comments and feedback until the end of August. To facilitate this process, we encourage you to share your insights primarily through our Discussion forum on GitHub. However, if you prefer, you can also send your feedback to us via email at [contact@spdci.org](mailto:contact@spdci.org).

Regards, 
Apoorva Sharma
